src/Symfony/Component/Intl/Languages.php

```diff
@@ -75,6 +75,11 @@ public static function getAlpha3Code(string $language): string
         return self::readEntry(['Alpha2ToAlpha3', $language], 'meta');
     }
 
+    public static function getAlpha3Codes(): array
```

Comment:
please add the phpdoc. We need at least `@return string[]` to document the return type more precisely.

And we need tests covering this API

Reply:
Done